pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -62,7 +63,7 @@ var (
 		"kubevirt_vmi_phase_count",
 		"VMI phase.",
 		[]string{
-			"node", "phase",
+			"node", "phase", "os", "workload", "flavor",
```

Comment:
Are those VM templates the standard for creating VMs? 

If I create a VM without the annotations, this will be the result:
```
kubevirt_vmi_phase_count{flavor="",node="node01",os="",phase="running",workload=""} 1
```

I ask because these labels will be created even for users who don't work with the templates, and thus, increase metric storage capacity.